orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/MonitorKatoTask.groovy

```diff
@@ -136,6 +148,9 @@ class MonitorKatoTask implements RetryableTask, CloudProviderAware {
       }
 
       if (stage.context."kato.task.retriedOperation" == true) {
+        Integer totalRetries = stage.context."kato.task.terminalRetryCount" as Integer
+        log.info("Completed kato task ${katoTask.id} after ${totalRetries} retries.")
```

Comment:
i think it would be useful to spit the exception that necessitated the retries (if there was one...) here - will make our log parsing easier